internal/mode/static/state/graph/gatewayclass.go

```diff
@@ -75,11 +76,27 @@ func buildGatewayClass(gc *v1beta1.GatewayClass) *GatewayClass {
 	}
 }
 
-func validateGatewayClass(gc *v1beta1.GatewayClass) error {
+func validateGatewayClass(gc *v1beta1.GatewayClass, npCfg *ngfAPI.NginxProxy) error {
 	if gc.Spec.ParametersRef != nil {
-		path := field.NewPath("spec").Child("parametersRef")
-		return field.Forbidden(path, "parametersRef is not supported")
+		if _, ok := supportedParamKinds[string(gc.Spec.ParametersRef.Kind)]; !ok {
+			path := field.NewPath("spec").Child("parametersRef")
```

Comment:
could we report path for the specific field?  so spec.parameterRef.Kind, rather than spec.ParameterRef.
(here and for the namespace)
could we make it consistent with https://github.com/nginxinc/nginx-gateway-fabric/blob/09fe18c212ecaaf11f6b114cc16739cdb1f0f15c/internal/mode/static/state/graph/gateway_listener.go#L348-L359 (error types)